src/main/java/frc/robot/subsystems/AlgaeArm/AlgaeArmIOPB.java

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.AlgaeArm;
+
+import com.ctre.phoenix6.configs.MotorOutputConfigs;
+
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+import com.revrobotics.RelativeEncoder;
+import com.revrobotics.spark.SparkBase.PersistMode;
+import com.revrobotics.spark.SparkBase.ResetMode;
+import com.revrobotics.spark.SparkLowLevel.MotorType;
+import com.revrobotics.spark.SparkMax;
+import com.revrobotics.spark.config.SparkBaseConfig.IdleMode;
+import com.revrobotics.spark.config.SparkMaxConfig;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class AlgaeArmIOPB implements AlgaeArmIO {
+
+  private final SparkMax armMotor = new SparkMax(Constants.PracticeBotConstants.ALGAE_ARM_ID, MotorType.kBrushless); // placeholder                                                                                                                    // ID
+  private final RelativeEncoder encoder = armMotor.getEncoder();
+
+  private final SparkMaxConfig sparkMaxConfig = new SparkMaxConfig();
+
+  /** Creates a new AlgaeArmIOPB. */
+  public AlgaeArmIOPB() {
+    sparkMaxConfig.idleMode(IdleMode.kBrake);
+    sparkMaxConfig.inverted(false);
+    armMotor.configure(sparkMaxConfig, ResetMode.kResetSafeParameters, PersistMode.kPersistParameters);
+  }
+
+  public void updateInputs(AlgaeArmIOInputs inputs) {
+    inputs.algaeArmPosition = encoder.getPosition();
+    inputs.algaeArmVoltage = armMotor.getBusVoltage() * armMotor.getAppliedOutput();
+  }
+
+  public void setDutyCycle(double dutyCycle) {
+    armMotor.set(dutyCycle);
+  }
+
+  public void setPosition(double position) {
+    encoder.setPosition(position);
```

Comment:
We need to use the built in PID on the spark max to control the position of the arm